datastreams/aggregator.go

```diff
@@ -47,6 +48,7 @@ type statsGroup struct {
 	parentHash     uint64
 	pathwayLatency *ddsketch.DDSketch
 	edgeLatency    *ddsketch.DDSketch
+	payloadSize    *ddsketch.DDSketch
```

Comment:
We can check Min/Max values, and if they both are 0 - do not create metric points. I'm not 100% sure it's worth doing, since one client may send 0s, another one may send non-zeros and dropping data in this case will be incorrect.